protobufs/generated/messages/events.proto

```diff
@@ -91,6 +91,684 @@ enum EventType {
   MEDICATION_STATEMENT__MEDICATION__PRE_SEARCH = 201;
   MEDICATION_STATEMENT__MEDICATION__POST_SEARCH = 202;
   MEDICATION_STATEMENT__MEDICATION__SELECTED = 203;
+
+  // Specific Command events
+
+  ADJUST_DIAGNOSIS_COMMAND__PRE_COMMAND_ORIGINATE = 1000;
```

Comment:
Can we change these to remove the second "COMMAND"? Seems redundant.

Example:

```suggestion
  ADJUST_DIAGNOSIS_COMMAND__PRE_ORIGINATE = 1000;
```

Reply:
@aduane I like it better if we remove the first COMMAND and keep the second one. The reason for this is that they match the naming of the canvas-core command events, which are named `PreCommandOriginate`, `PostCommandOriginate`, etc. How about `ADJUST_DIAGNOSIS__PRE_COMMAND_ORIGINATE`?